lua/starfall/libs_sh/holograms.lua

```diff
@@ -385,7 +385,11 @@ else
 			if bone ~= nil then checkluatype(bone, TYPE_BOOL) end
 			if attachment ~= nil or bone then
 				if isstring(attachment) then
-					attachment = bone and parent:LookupBone(attachment) or parent:LookupAttachment(attachment)
+					if bone then
+						attachment = parent:LookupBone(attachment) or 0
```

Comment:
or -1

Reply:
IIRC -1 was causing issues actually, making the holos disappear or teleport to world origin and not actually parent, will test it out quickly, give me a second

Reply:
Oh never mind, I see what you meant, the `if attachment < 0 or attachment > 255` will catch it and error